### PR TITLE
Add support for django 1.8 and 1.9

### PR DESCRIPTION
**Pull request by Ian-Foote**

### CODE REVIEW

**Comment by Ian-Foote** [comment]
@incuna/backend Review?

**Comment by adam-thomas** [comment]
HWTI

File: changelog.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog for `incuna-videos`
 
+## v7.0.0
+
+* Add support for Django 1.8 and 1.9.
+* Drop support for Django 1.6 and 1.7.
+* Update required versions of `FeinCMS` and `django-imagekit`.
```
Comment (meshy) [inline changelog.md]: dropping support for python 3.3?

File: .travis.yml [hunk 2]
```diff
@@ -9,13 +9,13 @@ env:
...
 install:
   - psql -c 'CREATE DATABASE videos' -U postgres;
-  - pip install -r test_requirements.txt --use-wheel
-  - pip install -e . --use-wheel
-  - pip install $DJANGO --use-wheel
-  - pip install coveralls --use-wheel
+  - pip install -r test_requirements.txt
+  - pip install -e .
+  - pip install $DJANGO
+  - pip install coveralls
```
Comment (KevinEtienne) [inline .travis.yml]: should `coveralls` be part of the requirements file

Comment (Ian-Foote) [inline .travis.yml]: It's only needed for travis right?

Comment (KevinEtienne) [inline .travis.yml]: I got confuse with `coveralls` and `coverage`..

**Pull request merged**